wrangler-api/src/main/java/io/cdap/wrangler/api/ErrorRecordBase.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2017-2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.wrangler.api;
+
+import java.util.List;
+
+/**
+ * Base class for error record that includes the critical fields.
+ */
+public class ErrorRecordBase {
+
+  // Message as to why the row errored.
+  protected final String message;
+  // Code associated with the message.
+  protected final int code;
+  protected final boolean showInWrangler;
+  protected final List<Object> causes;
+
+  public ErrorRecordBase(
+      String message, int code, boolean showInWrangler, List<Object> causes) {
```

Comment:
the CDAP related projects (including wrangler) use a checkstyle that only requires lines to be within 120 characters and not 80, so these could all be on the previous line.

Reply:
Done.